homework/src/index.js

```diff
@@ -30,17 +31,88 @@
     return elem;
   }
 
-  function main(url) {
-    fetchJSON(url, (err, data) => {
-      const root = document.getElementById('root');
-      if (err) {
-        createAndAppend('div', root, { text: err.message, class: 'alert-error' });
-      } else {
-        createAndAppend('pre', root, { text: JSON.stringify(data, null, 2) });
-      }
+  function renderRepositories(index, repositoriesdata, container) {
+    const repoInfo = createAndAppend('div', container, { 'class': 'left-div box' });
+    const table = createAndAppend('table', repoInfo);
+
+    const repoTr = createAndAppend('tr', table);
+    createAndAppend('td', repoTr, { 'text': 'Repository :', 'class': 'label' });
+    const repoName = createAndAppend('td', repoTr, { 'id': 'repo-name' });
+    const url = createAndAppend('a', repoName, { 'target': '_blank', });
+
+    const descriptionTr = createAndAppend('tr', table);
+    const descriptionName = createAndAppend('td', descriptionTr, { 'text': 'Description :', 'class': 'label' });
+    const descriptionDetails = createAndAppend('td', descriptionTr);
+
+    const forkTr = createAndAppend('tr', table);
+    createAndAppend('td', forkTr, { 'text': 'Forks :', 'class': 'label' });
+    const forkNum = createAndAppend('td', forkTr);
+
+    const updateTr = createAndAppend('tr', table);
+    createAndAppend('td', updateTr, { 'text': 'Updated :', 'class': 'label' });
+    const updateNum = createAndAppend('td', updateTr);
+
+    url.innerText = repositoriesdata[index].name;
+    url.setAttribute('href', repositoriesdata[index].html_url);
+    if (repositoriesdata[index].description === null) {
+      descriptionName.innerHTML = '';
+      descriptionDetails.innerText = '';
+    } else {
+      descriptionName.innerText = "Description: ";
+      descriptionDetails.innerText = repositoriesdata[index].description;
+    }
+    forkNum.innerText = repositoriesdata[index].forks;
+    const updateRepo = new Date(repositoriesdata[index].updated_at);
+    updateNum.innerText = updateRepo.toLocaleString("en-US");
```

Comment:
1. You could directly set the text in the `createAndAppend()` call.
2. If you use `.toLocaleString()` without an argument the default locale that the user selected in the operating system is used. For instance, in my computer I have set the default locale to use a European date and time format. You are explicitly setting it here to use the US format (e.g. using AM/PM suffixes).